artifact/sha/platform/openssl/sha.cpp

```diff
@@ -119,13 +99,10 @@ expected::ExpectedSize Reader::Read(vector<uint8_t> &dst) {
 		return 0;
 	}
 
-	if (EVP_DigestUpdate(sha_handle_.get(), buffer_.data(), bytes_read.value()) != 1) {
+	if (EVP_DigestUpdate(sha_handle_.get(), &*start, bytes_read.value()) != 1) {
```

Comment:
Nit: I'm sure you've got your bindings right, but can I still ask for a parenthesis?

Reply:
You mean `&(*start)`? Not sure if that makes it clearer.. I think I vote for keeping it as is.

Reply:
I could do `&start[0]`, although my personal preference is still the former. Let me know which you prefer.

Reply:
haha, suit yourself

Reply:
I would hugely prefer the `&start[0]` syntax. With emphasis on `I`

Reply:
Haha, ok, that settles it!